mindmeld/components/_config.py

```diff
@@ -899,3 +905,38 @@ def get_nlp_config(app_path=None, config=None):
         pass
 
     return _get_default_nlp_config()
+
+
+def _get_default_auto_annotator_config():
+    return copy.deepcopy(DEFAULT_AUTO_ANNOTATOR_CONFIG)
+
+
+def get_auto_annotator_config(app_path=None, config=None):
+    """Gets the automatic annotator config for the app at the
+    given path.
+
+    Args:
+        app_path (str, optional): The location of the MindMeld app
+        config (dict, optional): A config object to use. This will
+            override the config specified by the app's config.py file.
+            If necessary, this object will be expanded to a fully
+            specified config object.
+
+    Returns:
+        dict: The automatic annotator config.
+    """
+
+    if config:
+        return config
+    try:
+        auto_annotator_config = getattr(
+            _get_config_module(app_path),
+            "AUTO_ANNOTATOR_CONFIG",
+            DEFAULT_AUTO_ANNOTATOR_CONFIG,
+        )
+        return auto_annotator_config
+    except (OSError, IOError):
+        logger.info(
+            "No app configuration file found. Using the default automatic annotator config."
+        )
+        return _get_default_auto_annotator_config()
```

Comment:
Why do we need to have a separate function to return the deepcopy? Can we just do the same thing in line 935?